W-15312003 RPA Manager release notes 2024-03/2. (#350)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[march-25-2024]]
== March 25, 2024

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

> * RPA Manager now doesn't pause a run configuration when a process execution fails because an RPA Bot is not able to establish a secure session.

Reviewer: ```suggestion
* RPA Manager now doesn't pause a run configuration when a process execution fails because an RPA Bot can't establish a secure session.
```
